src/test/resources/alma-fix/990367761810206441.json

```diff
@@ -4,6 +4,21 @@
   "hbzId" : "HT020582812",
   "deprecatedUri" : "http://lobid.org/resources/HT020582812#!",
   "title" : "Sora kara mita nihon",
+  "alternateGraphicRepresentation" : [ {
+    "$append" : {
+      "script" : {
+        "id" : "https://unicode.org/iso15924/iso15924.txt#Jpan",
+        "label" : "iso15924.txt#Jpan"
+      }
+    },
+    "$last" : {
```

Comment:
@blackwinter can you identify why the resultsare not written in the same object and the script is separated?

Reply:
Not off the top of my head, no. I can look at it later.

Reply:
Also note that the object keys are wrong: `$append` and `$last`.

Reply:
The object keys seems to be the key problem. I narrowed down the problem: https://github.com/hbz/lobid-resources/pull/1854/commits/d510f56925aa32cbe25e6651e32a67e3d46bb1f5

It is not the merging of additional elements.
Also it is not the conditional or the list-bind:
```
do list(path:"alternateGraphicRepresentation[]","var":"$AGR")
              if in ("$880.@script.id","$AGR.script.id")
```

Reply:
You need to put `set_array("alternateGraphicRepresentation[]")` after any `vacuum()`, otherwise the empty array will be removed before it's used.

Reply:
thanks, i knew that using vacuum in the middle of the transformation was a bad idea. Will fix this.